plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergMaterializedViewTest.java

```diff
@@ -505,6 +505,16 @@ public void testDropMaterializedViewCannotDropTable()
         assertUpdate("DROP TABLE test_drop_materialized_view_cannot_drop_table");
     }
 
+    @Test
+    public void testDropMaterializedViewWithoutStorageTable()
+    {
+        assertUpdate("CREATE MATERIALIZED VIEW test_drop_materialized_view_without_storage_table AS SELECT * FROM base_table1");
+        String catalogName = getSession().getCatalog().orElseThrow();
+        SchemaTableName storageTable = getStorageTable(catalogName, "test_drop_materialized_view_without_storage_table");
+        assertUpdate(format("DROP TABLE %s", storageTable.getTableName()));
+        assertUpdate("DROP MATERIALIZED VIEW test_drop_materialized_view_without_storage_table");
```

Comment:
Please add a few more queries for other operations on MVs like REFRESH, DESCRIBE, listing of MVs through information_schema etc.